dbux-graph-host/src/graph/controllers/GraphNode.js

```diff
@@ -66,28 +66,32 @@ export default class GraphNode extends HostComponentEndpoint {
     }
 
     this.setOwnMode(mode);
-    // GraphNodeMode.switchCall(mode, this.modeHandlers);
 
     // propagate mode to sub graph
     const childMode = this.getChildMode();
     for (const child of this.owner.children) {
       if (!hasGraphNode(child)) {
-        this.logger.warn(`GraphNode owner's children contains no graphNode`);
+        this.logger.warn(`GraphNode owner's children ${child.debugTag} contains no graphNode`);
```

Comment:
This warning is better, but still not very helpful to a dev who does not currently work on this exact feature. Usually, you want to add a bit more context information: at least we usually want to add which function/operation/feature caused this.